fmriprep/cli/run.py

```diff
@@ -786,7 +786,18 @@ def build_workflow(opts, retval):
     boilerplate = retval['workflow'].visit_desc()
 
     if boilerplate:
-        (logs_path / 'CITATION.md').write_text(boilerplate)
+        # To please git-annex users and also to guarantee consistency
+        # among different renderings of the same file, first remove any
+        # existing one
+        citation_files = {
+            ext: str(logs_path / 'CITATION.' + ext)
```

Comment:
Ordering of operations, better keep pathlib.Path

```suggestion
            ext: logs_path / ('CITATION.%s' % ext)
```